py-polars/polars/datatypes/classes.py

```diff
@@ -415,6 +415,20 @@ def __repr__(self) -> str:
 class Categorical(DataType):
```

Comment:
This class needs an `__eq__` / `__hash__` / `__repr__` implementation like the other dtypes.